app/models/user.rb

```diff
@@ -22,4 +22,7 @@ class User < ApplicationRecord
   # NOTE: we will want to add confirmable later on. Will require sendgrid setup (or w/e client we use.)
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :authored_pair_requests, class_name: "PairRequest", foreign_key: "author_id"
+  has_many :received_pair_requests, class_name: "PairRequest", foreign_key: "invitee_id"
```

Comment:
for now let's add `dependent: :destroy`. We might want to allow orphans to retain data but not sure.